blocks/table/table.css

```diff
@@ -0,0 +1,22 @@
+.table {
+    width: 100%;
+    overflow-x: auto;
```

Comment:
Could you please add 2.4rem margin-top and bottom? let's align this to what images have.